app/src/main/java/ch/sdp/vibester/helper/AdapterHelper.kt

```diff
@@ -24,20 +26,26 @@ class AdapterHelper {
     }
 
     /**
-     * @param buttonId ID of the button
-     * @param imageId ID of the image
-     * @param itemView View contains the button and image
-     * change the view from button to image in itemView
```

Comment:
I find it's clear enough what this function and its parameter do, with the carefully chosen function's name and parameters' names.
Do you think they are needed? I can add them back if you insist.

Reply:
The thing is that IDE uses the @param for helping the user when coding. It's also used with web doc generator: https://www.geeksforgeeks.org/what-is-javadoc-tool-and-how-to-use-it/ explained here.
Then it may not change a lot for this project but if I were the project manager I would enforce them. (this is not the case, so it should be a team decision)
Also for a reason of coherence for the codebase.

Also as they were already here, I don't see the benefit of removing them.
Then this is my point of view but maybe ask others on Telegram.

I think for the final submission they might read our code. They would appreciate if it was well documented. I mean even for small getter() (even if it's trivial to understand their purpose) I think we should document it (all public interfaces) as we learned in the old java class. (then it's only my opinion, we should discuss as whole team)

Reply:
ok, I see. I also refactored the function a bit so I thought maybe we don't need the documentation anymore, but I will add them back.